recipes/scrapy/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "Scrapy" %}
+{% set version = "1.1.3" %}
+{% set hash_type = "md5" %}
```

Comment:
I believe that sha256 is preferred: https://github.com/conda-forge/staged-recipes/wiki/Frequently-asked-questions#2-how-do-i-populate-the-hash-field